packages/cloud_firestore/cloud_firestore/example/ios/Podfile

```diff
@@ -1,5 +1,5 @@
 # Uncomment this line to define a global platform for your project
-platform :ios, '12.0'
```

Comment:
Should we do that for all the other example iOS Podfile?
Also, it might be worth it to open them and update in XCode as so we don't have to do it each time we want to compile something on iOS?

Reply:
Yeah, good catch 👍